app/views/school_search/show.html.erb

```diff
@@ -14,10 +14,12 @@
         <p>
           <%= govuk_link_to "Clear filters", "?" %>
         </p>
-        <%= f.govuk_text_field(
-              :school_name,
-              value: @school_search_form.school_name,
-              label: {text: "School name or group", class: "govuk-label govuk-label--s"}) %>
+
+        <div class="govuk-form-group">
+          <label for="school-search-form-school-name-field" class="govuk-label govuk-label--s school-search-label">School name or group</label>
+          <div id="school-search-form-autocomplete" class="autocomplete-wrapper"></div>
+        </div>
```

Comment:
Interesting! Do we definitely want this here? Has it been specified?

Reply:
Good point - the ticket specified the proof of concept and that we might use it in other parts of the app - this was the most logical place as I could test whenever the params are being sent correctly and that it doesn't interfere with existing functionality

Reply:
@ethanmills are you ok with it?

Reply:
Ok for now. I think it will need to be removed from this page in the upcoming sprint, as finding schools is one of the things we'll be working on.

Reply:
👍